Experience/experience.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Resume & Honors!
+		</title>
+		<link rel="stylesheet" type="text/css" href="../style.css">
+	</head>
+	
+	<body>
+
+		<nav>
+			<a href="../index.html"> Home </a> |
+			<a href="experience/experience.html" target="_blank"> Resume and Honors</a> |
+			<a href="http://www.fcbarcelona.com/all-about-leo-messi" target="_blank"> Official Messi site</a> |
+			<a href="http://www.worldsoccershop.com/shop-by-player-messi--lionel.html" target="_blank"> Buy Gear </a>
+		</nav>
+
+		<h1>Messi Fever! </h1>
+
+		<img src="../messi experience.jpg" class="mainexp">
+
+		<h3 class="experience">FC Barcelona</h3>	
+		<p class="pexperience">
+			Winner:
+			<ul>
+				<li>La Liga or the League in Spanish: 2004-05, 2005-06, 2008-09, 2009-10, 2010-11, 2012-13, 	2014-15</li>
+				<li>Copa del Rey: 2008-09, 2011-12, 2014-15</li>
+				<li>UEFA Champions League : 2005-06, 2008-09, 2010-11, 2014-15</li>
+				<li>Supercopa de Espana : 2005, 2006, 2009, 2010, 2011, 2013</li>
+				<li>UEFA Super Cup : 2009, 2011, 2015</li>
+				<li>FIFA Club World Cup : 2009, 2011, 2015</li>
+			</ul>
+		</p>
+
+		<p class="pexperience">
+			Runner-up:
+			<ul>	
+				<li>La Liga: 2011-12, 2013-14</li>
+				<li>Copa del Rey: 2010-11, 2013-14</li>
+				<li>Supercopa de Espana: 2012, 2015</li>
+				<li>UEFA Super Cup: 2006</li>
+			</ul>
+		</p>
+
+			
+
+		<h3 class="experience">	Argentina </h3>
+		<p class="pexperience">
+			Winner:
+			<UL>
+				<li>Olympic Gold Medal: 2008</li>
+				<li>FIFA U-20 World Cup: 2005</li>
+
+			</UL>
+
+				Runner-up:
+			<ul>
+				<li>FIFA World Cup 2014 </li>
+				<li>Copa América: 2007, 2015 </li>
+			</ul>
+		</p>
+
+		<h3 class="individual">	Individual Awards </h3>
+		<h3 class="experience">	Barcelona: </h3>
+
+		<p class="pexperience">
+			<UL>	
```

Comment:
:hammer: Make sure to keep all elements lowercased. This `<ul>` is uppercased here.